portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsEntries.testcase

```diff
@@ -143,13 +143,54 @@ definition {
 	}
 
 	@description = "Verify that with Single Approver Workflow enabled and after the user Submit for Publication, the form entry will have a pending label"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test WorkflowSubmissionChangesLabelToPending {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
+		
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Text");
+
+		FormsAdminNavigator.gotoSettings();
+
+		Form.configureWorkflow(workflowDefinition = "Single Approver");
+
+		Form.save();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.openPublishURL(publishFormURL = "${publishedFormPublicURL}");
+
+
+		FormFields.editText(
+			fieldName = "Text",
+			fieldValue = "Test Text");
+
+		FormPortlet.submitLocalizedSuccessfully(buttonName = "Submit for Publication");
```

Comment:
Hey @PauloVAS, the last step planned for this test is `Assert the Success Page`, but this line is just asserting that a success message is being displayed. In light of this, can you please assert that the Success Page is also being displayed?

The macro `viewSuccessPage` will do it for you. :slightly_smiling_face: